pkg/ast/pipesearch/searchHandler.go

```diff
@@ -112,7 +115,7 @@ func ParseSearchBody(jsonSource map[string]interface{}, nowTs uint64) (string, u
 			startEpoch = uint64(val)
 		case string:
 			defValue := nowTs - (15 * 60 * 1000)
-			startEpoch = parseAlphaNumTime(nowTs, string(val), defValue)
+			startEpoch, err = parseAlphaNumTime(nowTs, string(val), defValue)
```

Comment:
You should check if `err != nil` and if so, log an error message and return early.

Same comment for lines 139, 248, 363

Reply:
Got it, will make the change

Reply:
Around lines 120 and 144 you should return early with the error immediately after emitting the error log. It looks like you're correctly early-returning at the other places I mentioned